src/scvmm/azext_scvmm/_params.py

```diff
@@ -290,3 +318,77 @@ def load_arguments(self: AzCommandsLoader, _):
             options_list=['--avset-name', '-a'],
             help="Name of the Availabilty Set.",
         )
+
+    with self.argument_context('scvmm vm guest-agent enable') as c:
+        c.argument(
+            'vm_name', help="Name of the VM."
+        )
+        c.argument(
+            'username',
+            options_list=['--username'],
+            help="Username to use for connecting to the vm.",
+        )
+        c.argument(
+            'password', options_list=['--password'],
+            help="Username password credentials to use for connecting to the VM.",
+        )
+        c.argument(
+            'https_proxy', help="HTTPS proxy server url for the VM.",
+        )
+
+    with self.argument_context('scvmm vm guest-agent show') as c:
+        c.argument(
+            'password', options_list=['--password'],
+            help="Username password credentials to use for connecting to the VM.",
+        )
+
+    with self.argument_context('scvmm vm guest-agent show') as c:
+        c.argument(
+            'vm_name', help="Name of the VM.",
+        )
+
+    with self.argument_context('scvmm vm extension list') as c:
+        c.argument('vm_name', type=str, help='The name of the vm containing the extension.')
+        c.argument(
+            'expand', help='The expand expression to apply on the operation.')
+
+    with self.argument_context('scvmm vm extension show') as c:
+        c.argument(
+            'vm_name', type=str, help='The name of the vm containing the extension.',
+            id_part='name')
+        c.argument('name', type=str, help='The name of the vm extension.', id_part='child_name_1')
+
+    for scope in ['scvmm vm extension update', 'scvmm vm extension create']:
+        with self.argument_context(scope) as c:
+            c.argument(
+                'vm_name', type=str, help='The name of the vm where the extension '
```

Comment:
It is not necessary to specify the `str` type. `str` is the default value for `type` field.
```suggestion
                'vm_name', help='The name of the vm where the extension '
```

Reply:
Removed type = str from all places in this file.